kubernetes/generateKeys.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+rm -rf keys
+mkdir -p keys
+
+cp ${HOME}/.ssh/id_rsa.pub keys/id_rsa.pub
+cp ${HOME}/.ssh/id_rsa keys/id_rsa
+
+cp keys/id_rsa.pub keys/authorized_keys
+
+kubectl delete secret ssh-keys | true
+
+kubectl create secret generic ssh-keys \
+--from-file=keys/id_rsa \
+--from-file=keys/id_rsa.pub \
+--from-file=keys/authorized_keys
```

Comment:
Could you apply the change from https://github.com/andreaTP/multi-node-k8s/pull/1/files ? ;)

Reply:
Changes are done here:
https://github.com/andreaTP/akka/commit/bba5460ec6eeb62fa13b5c83c78020c5ea014b86
and CI is running here:
https://github.com/andreaTP/akka/runs/3262773405?check_suite_focus=true

I'll cherry-pick if it turns green 👍

Reply:
Thanks @raboof ! verified:
https://github.com/andreaTP/akka/runs/3262773405?check_suite_focus=true